packages/airnode-examples/scripts/make-request.ts

```diff
@@ -34,33 +31,12 @@ const waitForFulfillment = async (requestId: string) => {
 
 const makeRequest = async (): Promise<string> => {
```

Comment:
TBH I don't like this change. It introduces a lot of duplicate code (which I get it's not such a big deal for examples but still) and you've removed imo important comments.
Wouldn't be better to create a separate `makeTemplateRequest` method and use that in that one specific example?

Reply:
> and you've removed imo important comments.

I've removed them, because previously it was a generic `make-request.ts` script that decided that the first endpoint is going to be used. This has now been moved to the individual `request-utils.ts` file of the integration.

> Wouldn't be better to create a separate makeTemplateRequest method and use that in that one specific example?

No, because there is a generic `make-request.ts` which encapsulates the common parts of the examples and it has to call the same method of examples.